web-app/templates/publication.html

```diff
@@ -0,0 +1,26 @@
+<div class="row">
+<div class="col-sm-2">
+    <ol class="list-inline">
```

Comment:
I think this looked better the way it was before, with the smaller link, green and on the right.